tests/integ/modin/series/test_astype.py

```diff
@@ -404,26 +404,39 @@ def test_python_datetime_astype_DatetimeTZDtype(seed):
 
 
 @sql_count_checker(query_count=1)
-def test_astype_int_to_timedelta():
-    native_series = native_pd.DataFrame(data=[12345678, 2])
-    snow_series = pd.DataFrame(native_series)
+def test_astype_numeric_to_timedelta():
+    native_series = native_pd.Series(data=[12345678, 2.6])
+    snow_series = pd.Series(native_series)
     eval_snowpark_pandas_result(
         snow_series, native_series, lambda series: series.astype("timedelta64[ns]")
     )
 
 
-@sql_count_checker(query_count=0)
-def test_astype_to_timedelta_negative():
-    snow_series = pd.Series(data=["1", "2"])
-    with pytest.raises(
-        TypeError, match=r"dtype object cannot be converted to timedelta64\[ns\]"
-    ):
-        snow_series.astype("timedelta64[ns]")
-    snow_series = pd.Series(data=[1.25, 3])
-    with pytest.raises(
-        TypeError, match=r"dtype float64 cannot be converted to timedelta64\[ns\]"
-    ):
-        snow_series.astype("timedelta64[ns]")
+@sql_count_checker(query_count=1)
+def test_astype_non_numeric_to_timedelta():
```

Comment:
ditto, this is numeric?

Reply:
No should also be non_numeric (i.e. `[1, "2"]`)